### PR TITLE
Add perks section

### PR DESCRIPTION
**Pull request by josh-**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -617,28 +617,51 @@ <h2>Add Ons</h2>
...
       <h1>Perks in Detail</h1>
       <div class="perks-row">
-        <div class="perks-col pic"></div>
-        <div class="perks-col">
-          WOW
+        <div class="perks-col pic">
+          <img src="./images/careers.jpg">
+        </div>
+        <div class="perks-col text">
+          <h1>Careers Fair and Careers Roundup</h1>
```
Comment (erfanio) [inline index.html]: can these be `h2`s to keep the hierarchy

File: index.css [hunk 2]
```diff
@@ -342,14 +349,23 @@ html {
...
+.perks-col.text {
   padding: 20px;
 }
 
 .perks-col.pic {
   background: #ccc;
 }
 
+.perks-col.pic img {
+  object-fit: cover;
```
Comment (erfanio) [inline index.css]: wow this is cool. never seen it

Comment (josh-) [reply index.css]: Yeah I know right! Learnt about it from Wes Bos 👌

File: index.html (hunk 1 above)
Comment (josh-) [reply index.html]: Yeah good call! Will do 🤙

**Pull request merged**